css/index.css

```diff
@@ -66,11 +66,47 @@ p {
     line-height: 1.4;
 }
 
+button {
+    margin-top: 2%;
+    color: black;
+    background-color: white;
+
```

Comment:
I'm not sure this button styling quite matches the design document. Maybe look into that and try to match it better, but that is really the only minor issue I see!

Reply:
I was lazy with the button design and just wanted to finish the challenge first, but I can look back and style the button a little better.